src/settings_type.h

```diff
@@ -108,6 +108,7 @@ struct GUISettings {
 	uint8  window_soft_limit;                ///< soft limit of maximum number of non-stickied non-vital windows (0 = no limit)
 	ZoomLevel zoom_min;                      ///< minimum zoom out level
 	ZoomLevel zoom_max;                      ///< maximum zoom out level
+	ZoomLevel sprite_zoom_min;               ///< minimum zoom out level for sprites
```

Comment:
Isn't this rather a "maximum zoom-in" level?
How about: "Maximum GRF-provided zoom-level to use. Force scaling for further zoom level."

Reply:
A good point - should we change the comment for `zoom_min` as well?